packages/react-static/src/static/webpack/webpack.config.prod.js

```diff
@@ -165,18 +164,29 @@ export default function(state) {
   result.devtool = false
   result.externals = [
     new RegExp(`${paths.PLUGINS}`),
+    // Externalise all node_modules except for react-static files that need to be bundled
     (context, request, callback) => {
-      const resolved = path.resolve(context, request)
+      // All paths starting with `./` or `/` may be bundled as usual
+      if (/^[./\\]/.test(request)) {
+        return callback();
+      }
+
+      // Attempt to resolve the requested module or file from `context` which points at
+      // the file's location where the dependency on `request` is located at.
+      const res = resolveFrom(`${context}/`, request);
+
       if (
-        [/react-static(\\|\/)lib(\\|\/)browser/, /webpack-flush-chunks/].some(
-          d => d.test(resolved)
-        )
+        res &&
+        // If the request is in node_modules and a js file it should be externalised,
+        /node_modules[/\\].*\.js$/.test(res) &&
+        // unless it's a react-static dependency
+        !/node_modules[/\\]react-static/.test(res)
```

Comment:
After https://github.com/react-static/react-static/pull/1444, we'd need to update this like so:

```suggestion
        !/node_modules[/\\]react-static/.test(res) &&
        // react-universal-component must be bundled for static export to work
        // https://github.com/react-static/react-static/issues/1446
        !/node_modules[/\\]react-universal-component/.test(res)
```

Part of me think it might be cleaner to put it in its own `if` statement ?

Reply:
Yeah I think we just need th entire list next.js is doing too